infrastructure/prod.tfvars

```diff
@@ -1,9 +1,9 @@
 infrastructure_env = "prod"
 
-evidence_submission_info_link = "https://track-appeal.platform.hmcts.net/evidence/appeal_id"
-sscs_manage_emails_link = "https://track-appeal.platform.hmcts.net/manage-email-notifications/mac"
-sscs_track_your_appeal_link = "https://track-appeal.platform.hmcts.net/trackyourappeal/appeal_id"
-hearing_info_link = "https://track-appeal.platform.hmcts.net/abouthearing/appeal_id"
-claiming_expenses_link = "https://track-appeal.platform.hmcts.net/expenses/appeal_id"
+evidence_submission_info_link = "https://www.track-benefit-appeal.service.gov.uk/evidence/appeal_id"
```

Comment:
you can use one env var to store the url and reuse that env for all the links as all of the use the same. Next time we need to change it it's only one single line change. ;)